web/admin/src/App.js

```diff
@@ -451,8 +453,10 @@ class App extends PureComponent {
         }
       }
     })
-    this.setState({ exporting: true, exportList: parsedData })
-    setTimeout(() => this.setState({ exporting: false }), 3000)
+    if (parsedData.length) {
```

Comment:
Will this result in the user being able to click "Export" but have nothing happen (if there is no data)?

Reply:
Correct. This is a really rare scenario where not a single scan has been completed. I could try to disable the button but that creates its own challenges because the info I know this.state.scansPerUserPerCategory could have blank info if set as an admin but no scans.

Reply:
👍 then we should have an `else` for this `if` to at least alert that there is nothing to export